src/rt/backtrace/dwarf.d

```diff
@@ -86,7 +87,24 @@ int traceHandlerOpApplyImpl(const void*[] callstack, scope int delegate(ref size
 
         if (locations.length > 0 && locations[i].line != -1)
         {
-            appendToBuffer("%.*s:%d ", cast(int) locations[i].file.length, locations[i].file.ptr, locations[i].line);
+            if (locations[i].directory.length == 0 || locations[i].directory[$ - 1] == '/')
+            {
+                appendToBuffer(
+                    "%.*s%.*s:%d ",
```

Comment:
Please just conditionally assign the strings to a temporary rather than duplicate the entire call.